code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1235,3 +1235,83 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "glass_yellow"
 	glass_name = "Eggnog"
 	glass_desc = "For enjoying the most wonderful time of the year."
+
+// IPC ALCHOHOL WOOOOOOOOOOO
+/datum/reagent/consumable/ethanol/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+	process_flags = SYNTHETIC
+	boozepwr = 20
+	glass_icon_state = "synthanolglass"
+	glass_name = "Glass of Synthanol"
+	glass_desc = "The equivalent of alcohol for synthetic crewmembers. They'd find it awful if they had tastebuds too."
+	taste_description = "motor oil and grease"
+
+/datum/reagent/consumable/ethanol/synthanol/robottears
+	name = "Robot Tears"
+	id = "robottears"
+	description = "An oily substance that an IPC could technically consider a 'drink'."
+	reagent_state = LIQUID
+	color = "#363636"
+	boozepwr = 10
+	glass_icon_state = "robottearsglass"
+	glass_name = "Glass of Robot Tears"
+	glass_desc = "No robots were hurt in the making of this drink."
+
+/datum/reagent/consumable/ethanol/synthanol/trinary
+	name = "Trinary"
+	id = "trinary"
+	description = "A fruit drink meant only for synthetics, however that works."
+	reagent_state = LIQUID
+	color = "#adb21f"
+	boozepwr = 50
+	glass_icon_state = "trinaryglass"
+	glass_name = "Glass of Trinary"
+	glass_desc = "Colorful drink made for synthetic crewmembers. It doesn't seem like it would taste well."
+
+/datum/reagent/consumable/ethanol/synthanol/servo
+	name = "Servo"
+	id = "servo"
+	description = "A drink containing some organic ingredients, but meant only for synthetics."
+	reagent_state = LIQUID
+	color = "#5b3210"
+	boozepwr = 50
+	glass_icon_state = "servoglass"
+	glass_name = "Glass of Servo"
+	glass_desc = "Chocolate - based drink made for IPCs. Not sure if anyone's actually tried out the recipe."
+
+/datum/reagent/consumable/ethanol/synthanol/uplink
+	name = "Uplink"
+	id = "uplink"
+	description = "A potent mix of alcohol and synthanol. Will only work on synthetics."
+	reagent_state = LIQUID
+	color = "#e7ae04"
+	boozepwr = 30
+	glass_icon_state = "uplinkglass"
+	glass_name = "Glass of Uplink"
+	glass_desc = "An exquisite mix of the finest liquoirs and synthanol. Meant only for synthetics."
+
+/datum/reagent/consumable/ethanol/synthanol/synthnsoda
+	name = "Synth 'n Soda"
+	id = "synthnsoda"
+	description = "The classic drink adjusted for a robot's tastes."
+	reagent_state = LIQUID
+	color = "#7204e7"
+	boozepwr = 50
+	glass_icon_state = "synthnsodaglass"
+	glass_name = "Glass of Synth 'n Soda"
+	glass_desc = "Classic drink altered to fit the tastes of a robot. Bad idea to drink if you're made of carbon."
```

Comment:
Pretty sure some of the IPC parts are made of "carbon-fibre".

Maybe say "meat" instead of "carbon".